backend/pom.xml

```diff
@@ -22,6 +22,39 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <optional>true</optional>
+        </dependency>
```

Comment:
If you add lombok, I'm pretty sure that you need to add an exclusion to the `spring-boot-maven-plugin`. Have a look on start.spring.io.